bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,512 @@
+<?xml version="1.0" ?>
+<thing:thing-descriptions bindingId="wundergroundupdatereceiver"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="last-received-datetime">
+		<item-type>DateTime</item-type>
+		<label>Last Received</label>
+		<description>The date and time of the last update.</description>
+		<category>Time</category>
+		<tags>
+			<tag>Point</tag>
+			<tag>Timestamp</tag>
+		</tags>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS"/>
+	</channel-type>
+
+	<channel-type id="datetime-utc">
+		<item-type>String</item-type>
```

Comment:
convert to DateTime instead of string?

Reply:
The string "now" is a valid value according to the documentation at wunderground.com and the value is generated by the weatherstation. The measured timestamp is is in lastRecieved.

Reply:
I see.
For the user it means that the time stamp generated by the station can be read (and converted) from datetime-utc, but if it is `now` lastRecieved must be checked. First provides UTC time, second is in in current time zone.
From my perspective it would be better if the binding abstracts this complexity, but I also agree that this is a feature or at least an improvement.
It would be possible to provide `String:datetime-utc` and `DateTime:datetime` (for convinience in local time zone) in parallel.

The following should somehow be possible in function `updateChannelState`:
`(unit == DateTime && state == "now") ? now() : new DateTime(state)`

Reply:
Agreed. I will consider how to do this transparently, but in the current iteration the type of dateutc needs to be String to support 'now'. A solution would be to remove the timestamp channel and convert in the proposed fashion by parsing the UTC datetime to LocalDateTime and only saving that. That of course would break any rules depending on both values existing. How does the community view this sort of change in a binding?
The weather station could also be mistaken about what the correct time is and submit incorrect datetime values, although that is probably not really an issue. If someone has read the weather.com documentation they could be confused about where the 'now' is disappearing.

Reply:
I'd convert the String to DateTime. If you want to do it in a future iteration, you could remove this channel completly for now, otherwise it would break backward compatibility.

Reply:
It's been added as an extra synthetic computed channel.